src/main/java/org/openstreetmap/atlas/locale/IsoCountry.java

```diff
@@ -135,6 +183,36 @@ else if (countryCode.length() == ISO3_LENGTH
         return Optional.empty();
     }
 
+    /**
+     * Provides IsoCountry for a valid country display name. Ignores capitalization (e.g. "united
+     * stAtes" and "United States" are the same)
+     *
+     * @param displayCountry
+     *            the display country name, e.g. "United States"
+     * @return an Optional containing the IsoCountry if present
+     */
+    public static Optional<IsoCountry> forDisplayCountry(final String displayCountry)
+    {
+        if (displayCountry != null)
+        {
+            /*
+             * We want to allow the displayCountry parameter to have inconsistent case. E.g.
+             * displayCountry="united States" should match IsoCountry<"United States">
+             */
+            String foundKey = null;
+            for (final String key : DISPLAY_COUNTRY_TO_ISO2.keySet())
+            {
+                if (displayCountry.equalsIgnoreCase(key))
+                {
+                    foundKey = key;
+                    break;
+                }
+            }
+            return Optional.ofNullable(ISO_COUNTRIES.get(DISPLAY_COUNTRY_TO_ISO2.get(foundKey)));
```

Comment:
scope nit:  Using the optional interface a bit ```Optional.ofNullable(foundKey).map(DISPLAY_COUNTRY_TO_ISO2::get).map(ISO_COUNTRIES::get)```

Reply:
Sure, why not?